index.js

```diff
@@ -1137,34 +1146,30 @@ module.exports = function (log, indexesPath) {
             startSeq + rawLength,
             limit - seqs.length,
             seqs,
-            (err, seqs) => ensureEnoughResults(err, startSeq + rawLength, seqs)
+            (e, seqs) => ensureEnoughResults(e, startSeq + rawLength, seqs)
           )
-        else processResults(err, seqs, resultSize, cb)
+        else processResults(seqs, resultSize)
```

Comment:
`processResults` wasn't using the last argument :|

Reply:
Ah, that was from when I had those functions a bit further out but realized that I was passing a ton of arguments to each function. Good find.